src/PrestaShopBundle/Resources/config/services/core/kpi.yml

```diff
@@ -21,3 +21,11 @@ services:
           - '@prestashop.adapter.kpi.empty_categories'
           - '@prestashop.adapter.kpi.top_category'
           - '@prestashop.adapter.kpi.average_products_in_category'
+
+    prestashop.core.kpi_row.factory.customers:
+        class: PrestaShop\PrestaShop\Core\Kpi\Row\KpiRowFactory
+        arguments:
+            - '@prestashop.adapter.kpi.most_common_customers_gender'
```

Comment:
I'm wondering if those adapter services should be in the core
maybe it would me more correct to declare aliases like:

```
    prestashop.core.kpi.most_common_customers_gender: '@prestashop.adapter.kpi.most_common_customers_gender'
```

so that core services fiels only point to core services, and we will only have to redeclare the proper service in the core later?

(not necessarily relevant for this specific PR)

Reply:
i dont know really.. :confused:  @matks wdyt?